### PR TITLE
Re-arrange members, Enno as head right column

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: team/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: page
 title: Team
 ---
 
-<h2>Core Akka Team</h2>
+<h2>Akka Team</h2>
```
Comment (patriknw) [inline team/index.md]: I think the reason we have said core team is that we also count community contributors as part of the team. Maybe we should even say Lightbend Akka Team here

**Comment by renatocaval** [comment]
@ennru, are you saying that we should stay as before and close this PR?

File: team/index.md (hunk 1 above)
Comment (renatocaval) [reply team/index.md]: I see. I will change it if we proceed with this PR

**Comment by ennru** [comment]
That's what I prefer.

**Comment by renatocaval** [comment]
I fine with that. I did that per discussion here: https://github.com/akka/akka.github.com/pull/676#issuecomment-584006585

Anyway, closing it...

**Pull request closed without merging**